docs/library/IPs/template_ip/index.rst

```diff
@@ -0,0 +1,119 @@
+:orphan:
+
+.. _template_ip:
```

Comment:
Maybe we should give it a name like tb_template_ip, as we already have this name in the HDL documentation. I'm not sure if it will come to a point where the tool gets confused

Reply:
It's okay to reuse tags, because when cross-referencing between repos we use `:ref-<repo>:` instead of just `:ref:`, e.g. ```:ref-hdl:`template_ip` ```